cmake/OpenCL.cmake

```diff
@@ -30,13 +30,16 @@ endif ()
 
 if (ENABLE_OPENCL)
     target_compile_definitions (qrack PUBLIC ENABLE_OPENCL=1)
+    target_compile_definitions (qrack PUBLIC CL_HPP_TARGET_OPENCL_VERSION=200)
 
     # Include the necessary options and libraries to link against
     target_include_directories (qrack PUBLIC ${PROJECT_BINARY_DIR} ${OPENCL_INCLUDE_PATH})
     target_compile_options (qrack PUBLIC ${OPENCL_COMPILATION_OPTIONS})
     target_link_libraries (unittest ${LIB_OPENCL})
     target_link_libraries (benchmarks ${LIB_OPENCL})
+endif (ENABLE_OPENCL)
 
+if (ENABLE_OPENCL)
```

Comment:
Why do this split when both are r same clause?

Reply:
Earlier versions had two "if" statements. Removed.